plugin-dev/Config/FilterPlugin.ini

```diff
@@ -7,10 +7,5 @@
 ;    /Extras/...
 ;    /Binaries/ThirdParty/*.dll
 /Scripts/*
-/LinuxIntermediate/Build/Linux/UnrealEditor/Inc/Sentry/UHT/*
-/LinuxIntermediate/Build/Linux/UnrealGame/Development/Sentry/*
-/LinuxIntermediate/Build/Linux/UnrealGame/Inc/Sentry/UHT/*
-/LinuxIntermediate/Build/Linux/UnrealGame/Shipping/Sentry/*
-/LinuxIntermediate/Build/Linux/x64/UnrealGame/Development/Sentry/*
-/LinuxIntermediate/Build/Linux/x64/UnrealGame/Shipping/Sentry/*
+/Gradle/*
```

Comment:
Is that change intentional? The how do the `LinuxIntermediate` related to `Gradle`?

Reply:
Yes, aside from adding `Gradle` here I've also removed `LinuxIntermediate` stuff which is now redundant (probably I've missed deleting these entries earlier). Should we tackle this kind of clean-up in a separate PR?